templates/digital-asset-template/frontend/pages/Mint/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { WalletSelector } from "@aptos-labs/wallet-adapter-ant-design";
+import "@aptos-labs/wallet-adapter-ant-design/dist/index.css";
+import { config } from "@/config";
+import { Image } from "@/components/ui/image";
+import { indexer } from "@/graphql/indexer";
+import { useQuery } from "@tanstack/react-query";
+import { BannerSection } from "./components/BannerSection";
+import { HeroSection } from "./components/HeroSection";
+import { StatsSection } from "./components/StatsSection";
+import { OurStorySection } from "./components/OurStorySection";
+import { HowToMintSection } from "./components/HowToMintSection";
+import { OurTeamSection } from "./components/OurTeamSection";
+import { FAQSection } from "./components/FAQSection";
+
+const COLLECTION_ADDRESS =
+  "0x7b6209587a6d298676e074753dfac2277a4b22a15beefab30111cc15c853c8f9";
+
+export function Mint() {
+  const { data, isLoading } = useQuery({
+    queryKey: ["app-state"],
+    refetchInterval: 1000 * 30,
+    queryFn: async () => {
+      const res = await indexer.TokenQuery({
+        collection_id: COLLECTION_ADDRESS,
+      });
+
+      const collection = res.current_collections_v2[0];
+      if (!collection) return null;
+
+      return {
+        maxSupply: collection.max_supply ?? 0,
+        totalMinted: collection.current_supply ?? 0,
+        uniqueHolders:
+          res.current_collection_ownership_v2_view_aggregate.aggregate?.count ??
+          0,
+        collection,
+        tokens: res.current_token_datas_v2,
+      };
+    },
+  });
+
+  if (isLoading) return <p>Loading...</p>;
+  if (!data) return <p>Collection not found</p>;
+
+  return (
+    <div style={{ overflow: "hidden" }} className="overflow-hidden">
```

Comment:
you are missing this part https://github.com/aptos-labs/create-aptos-dapp/pull/42/files#diff-5b6f10473a0d73d92cf4cd53852472aa6a3bdb2dff608ff8b6b084ea271dbf51L37-L44

also, we can get everything into a Header component